### PR TITLE
fix(http): change /query to use org/orgID

### LINKED ISSUE #2106: api: /api/v2/query uses `organization` and `organizationID` parameter but swagger has `org`
https://github.com/influxdata/platform/blob/master/http/requests.go#L11-L14

This should be `org` and `orgID`.

Additionally, orgID needs to be added to swagger:
https://github.com/influxdata/platform/blob/master/http/swagger.yml#L2489-L2493

### PR DESCRIPTION
**Pull request by goller**
Closes #2106 

_Briefly describe your proposed changes:_
We should use `org` and `orgID` rather than `organization` and `organizationID` as HTTP query parameters to `/api/v2/query`

_What was the problem?_
Documentation in swagger had `org`.

_What was the solution?_
Update docs and add tests to show org/organizationID,

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass
  - [x] swagger.json updated (if modified Go structs or API)
  - [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: http/requests_test.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+								ID: platform.ID(1),
+							}, nil
+						}
+						return nil, fmt.Errorf("unknown ID")
+					},
+				},
+			},
+		},
+		{
+			name:    "bad id returns errorn",
```
Comment (leodido) [inline http/requests_test.go]: typo (`errorn) ?

File: http/requests_test.go [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+				r:   httptest.NewRequest(http.MethodPost, "/api/v2/query?org=org1", nil),
+				svc: &mock.OrganizationService{
+					FindOrganizationF: func(ctx context.Context, filter platform.OrganizationFilter) (*platform.Organization, error) {
+						if *filter.Name == "org1" {
+							return &platform.Organization{
+								ID:   platform.ID(1),
+								Name: "org1",
+							}, nil
+						}
+						return nil, fmt.Errorf("unknown org name")
```
Comment (kelwang) [inline http/requests_test.go]: use &platform.Error{
     Code: platform.EInvalid,
     Msg: "unknown org name",
}

File: http/requests_test.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+					},
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := queryOrganization(tt.args.ctx, tt.args.r, tt.args.svc)
+			if (err != nil) != tt.wantErr {
```
Comment (kelwang) [inline http/requests_test.go]: compare error code, and http status code

Comment (goller) [reply http/requests_test.go]: Hey @kelwang, in this case this is a helper function that extracts information from the *http.Request, but, does not write a response:

https://github.com/influxdata/platform/blob/5297d1fee9f156af962700545b7ce0cc24a3be22/http/requests.go#L18

I'll check the error for its type.

**Pull request merged**